bundles/plugins/org.bonitasoft.studio.common/src/org/bonitasoft/studio/common/properties/AbstractBonitaDescriptionSection.java

```diff
@@ -14,6 +14,8 @@
  */
 package org.bonitasoft.studio.common.properties;
 
+import static org.eclipse.jface.layout.GridDataFactory.fillDefaults;
```

Comment:
using static is not very recommended http://www.javapractices.com/topic/TopicAction.do?Id=195 unless for constants and Tests assertions